src/Cake.NuGet/NuGetPackageInstaller.cs

```diff
@@ -175,6 +160,38 @@ public IReadOnlyCollection<IFile> Install(PackageReference package, PackageType
             return result;
         }
 
+        private static DirectoryPath GetPackagePath(DirectoryPath root, PackageReference package)
+        {
+            if (package.Parameters.ContainsKey("version"))
+            {
+                var version = package.Parameters["version"].First();
+                return root.Combine($"{package.Package}.{version}");
```

Comment:
> Yeah, but internally, NuGet CLI uses NuGetVersion for this that will have the same representation regardless of trailing zeros or not.

`nuget install ReportGenerator -Version 2.5.8` yields
![image](https://user-images.githubusercontent.com/1647294/28784664-564f2dbe-7614-11e7-8527-9db782c3345a.png)

So don't think so. it's nuspec says
```xml
<version>2.5.8</version>
```